test.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Comment
+ */
+
+echo 'Hello World!';
+
+$a = function_call();
+
+
+class TestFile {
+  /**
+   * Constant
+   *
+   * @var bool
+   */
+  const A = false;
+
+  /**
+   * Property
+   *
```

Comment:
⚠️ **[phpcs]** <WordPress.WhiteSpace.PrecisionAlignment.Found> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Found precision alignment of 2 spaces.